docs/quick-start/node-sdk-hello-world.md

```diff
@@ -0,0 +1,266 @@
+---
+id: hi-kin-node
+title: Hello World Node
+---
+
+With the Kin SDK for javascript Node, you can create a server back-end to support your client apps and integrate Kin.
+If you’re just getting started with Kin ecosystem, we suggest you spend a few minutes reading this [overview of the Kin architecture.](../kin-architecture-overview.md)
+
+The following tutorial is meant to run on a server and will serve as a crash course on the basic functionalities of the Kin SDK for Node.
+As you implement Kin in your service, you should wrap this SDK in your back-end services as you see fit.
+Here we will simply look at the most basic functions of the Kin SDK for Node showing you how to create an account on the Kin Blockchain, execute a transaction and read the user's current balance.
+
+This tutorial operates in the Kin Playground environment where you can develop and test your Kin integration.
+
+This tutorial is designed to take you from start to finish in 5 minutes or less, so get ready!
+
+## Setup
+First of all, you need to install the Kin SDK. The Kin SDK requires Python 3 and is installed with pip:
+
+```bash
+npm install @kinecosystem/kin-sdk
+```
+## Code Walkthrough
+This tutorial covers the primary elements of the script. See the Download link at the end of the article if you want to skip to the finished work.
+
+The script simply executes a set of predefined commands. No user interaction is required except for the creation of a public address.
+
+### The Basics
+With the Kin SDK for Node installed, you can create the simple `index.js` script. Let's import `kin`.
+
+```node
+// This is the Kin SDK
+import * as KinSdk from @kinecosystem/kin-sdk;
+
+```
+TODO: CHECH THE CODE ^^
+
+### Manage Accounts
+Let's create two accounts: one for a user of your service and one to receive some Kin from your user.
+
+Here you instantiate the `KinClient` and select in which environment you want to work. In this case, you'll work in the Playground (the test environment).
+
+```node
+  console.log('First we will create our KinClient object, and direct it to our test environment');
+  let client = new KinClient(integEnv);
```

Comment:
Just me but shouldn't we call this our Test Environment not integEnv?

Reply:
What's "integEnv"?